app/src/main/java/com/github/studydistractor/sdp/account/UserInterface.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.studydistractor.sdp.account
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+
```

Comment:
Also document the purpose of the interface ! You did document everything else in this class :(